tests/app/Infrastructure/Controller/GetsWalletCryptocurrenciesControllerTest.php

```diff
@@ -68,11 +40,8 @@ public function ifWalletIdExistsGetCoins()
         ];
         $wallet = new Wallet($walletId);
 
-        $this->walletDataSource
-            ->expects("findById")
-            ->with($walletId)
-            ->andReturn($wallet);
-
+        Cache::shouldReceive('has')->andReturn(true);
+        ;
```

Comment:
?????
```suggestion
        
```

Reply:
Wow, so strange it hasn't messed with that 🤣